WrightTools/units.py

```diff
@@ -152,48 +138,50 @@ def get_symbol(units) -> str:
     string
         LaTeX formatted symbol.
     """
-    if kind(units) == "energy":
-        d = {}
-        d["nm"] = r"\lambda"
-        d["wn"] = r"\bar\nu"
-        d["eV"] = r"\hslash\omega"
-        d["Hz"] = r"f"
-        d["THz"] = r"f"
-        d["GHz"] = r"f"
-        return d.get(units, "E")
-    elif kind(units) == "delay":
+    quantity = 1 * ureg[units]
+    if quantity.check("[length]"):
+        return r"\lambda"
+    elif quantity.check("1 / [length]"):
+        return r"\bar\nu"
+    elif quantity.check("[energy]"):
+        return r"\hslash\omega"
+    elif quantity.check("1 / [time]"):
+        return "f"
+    elif quantity.check("[time]"):
         return r"\tau"
-    elif kind(units) == "fluence":
+    elif quantity.check("[fluence]"):
         return r"\mathcal{F}"
-    elif kind(units) == "pulse_width":
-        return r"\sigma"
-    elif kind(units) == "temperature":
-        return r"T"
+    elif quantity.check("[temperature]"):
+        return "T"
+    elif ureg[units] in (ureg.deg, ureg.radian):
+        return r"\omega"
     else:
-        return kind(units)
+        return None
```

Comment:
this method may be cleaner as a dict... (of dimensionality -> str, with angle/other dimensionless units handled separately)

For _right now_ I'm inclined to keep it in the if/elif structure until and unless we decide to add more to it.
But I'm open to suggestions of what to add, and if given will likely refactor.

Reply:
I like leaving it as an if elif tree